WalletWasabi/JsonConverters/LabelJsonConverter.cs

```diff
@@ -0,0 +1,30 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using WalletWasabi.Models;
+
+namespace WalletWasabi.JsonConverters
+{
+	public class LabelJsonConverter : JsonConverter
```

Comment:
```suggestion
	public class SmartLabelJsonConverter : JsonConverter
```

Reply:
Thanks, I did it manually instead of accepting your commit suggestions, to make sure I won't miss out anything.